UserLeapKit.xcframework/ios-arm64_x86_64-simulator/UserLeapKit.framework/sprig_web_assets.bundle/snippet.js

```diff
@@ -20,12 +20,17 @@ function configure(environmentId, mobileHeadersJSON) {
 }
 
 function handleSurveyCallback(surveyState, callbackId) {
+    // no survey handler but the survey is shown
+    if (!callbackId) {
+        Sprig.dismissActiveSurvey()
```

Comment:
nit: missing ;

Reply:
will roll the nits into the next release